### PR TITLE
NSM - add sensor id to alert details

### PR DESCRIPTION
**Pull request by liorblob**
Need to test as soon as we get alerts

### CODE REVIEW

**Comment by anara123** [comment]
@liorblob still no alerts?

**Comment by liorblob** [comment]
@anara123 Nope, @ariansvirsky is anyone working on this?

File: Integrations/integration-McAfeeNSM.yml [hunk 1]
```diff
@@ -162,28 +163,72 @@ script:
...
+            Type: entryTypes.note,
+            Contents: alert,
+            ContentsFormat: formats.json,
+            ReadableContentsFormat: formats.markdown,
+            HumanReadable : (tableToMarkdown(title, md, headers) +
+                tableToMarkdown('Reference', reference) +
+                tableToMarkdown('Platform Affected', {Protocols : alert.description.protocals})
+            ),
+            EntryContext : {
+            'NSM.Alerts(val.ID && val.ID == obj.ID)' : createContext(context),
```
Comment (Itay4) [inline Integrations/integration-McAfeeNSM.yml]: `===`

File: Integrations/integration-McAfeeNSM.yml [hunk 2]
```diff
@@ -162,28 +163,72 @@ script:
...
+            'Attack SubCategory' : alert.description.attackSubCategory,
+            Comments : alert.description.comments.comments,
+        };
+        var links = alert.description.reference.additionInfo.split('<BR>');
+        for (var i in links) {
+            links[i] = '[{0}]({0})'.format(links[i]);
+        }
+
+        var reference = {
+            // 'arachNidsId': alert.description.reference.arachNidsId,
```
Comment (Itay4) [inline Integrations/integration-McAfeeNSM.yml]: remove

**Pull request merged**